.devcontainer/devcontainer.json

```diff
@@ -18,7 +18,7 @@
 			"terragrunt": "latest"
 		},
 		"ghcr.io/devcontainers/features/go:1": {
-			"version": "latest"
+			"version": "1.22.2"
```

Comment:
it'd be better if we use the same version as we have in the `go.mod` file (`1.22.3`)

Reply:
Done.